README.md

```diff
@@ -1,28 +1,50 @@
-# First Mobile Version
 
-> Description the project.
-I choosed the frist template in Figma and modify a lot styles. This steps is very important for me to lear daily a lot things in the wordl of programming. 
+# My first project in Microverse!
+
 
+> Description the project.
+  I have completed my project in compliance with the established guidelines. I am very happy with the changes made and I hope to continue learning day by day so that I can do projects like this and better ones. The project contains two main files, one HTML and one CSS. Also, added the linters, to make quality code.
 
 ## Built With
 
--HTML/CCS.LINTERS
+- HTML
+- CSS
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
 
+### Install
 
+### Usage
 
-👤 **Lucca MIcale**
+### Run tests
+
+### Deployment
+
+
+
```

Comment:
Great job @luccamicale 🤝

- [ ] You may kindly remove these unused sections from your readme file. You will need to add them later along as they become necessary.

Reply:
Good job following the suggestions of the previous reviewer